### PR TITLE
Fix troubleshooting test_venv.

### PR DESCRIPTION
**Pull request by itsayellow**
* [ ] I have added an entry to `docs/changelog.md`

## Summary of changes
Follow-up to PR #569 .  Fixing the "Try it with pip" section to not include activate / deactivate, and specify different Windows instructions, as @cs01 recommended.

## Test plan

Tested by running
```
# command(s) to exercise these changes
```

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -87,10 +87,16 @@ This is a tip for advanced users.  An easy way to check if pipx is the problem
...
 python3 -m venv test_venv
-source test_venv/bin/activate
-python3 -m pip install <problem-package>
+test_venv/bin/python3 -m pip install <problem-package>
+```
+
+### Windows
+```
+python -m venv test_venv
+test_venv/Scripts/python -m pip install <problem-package>
```
Comment (cs01) [inline docs/troubleshooting.md:99]: Looks great! I'm not sure, but I think one windows it needs to be `python.exe`?

Comment (itsayellow) [reply docs/troubleshooting.md:99]: I believe I actually tried on my VM Windows...but now you make me want to make sure and try it again 😆 

I think like regular entry points on windows, you don't need to type the `.exe`, but I will check again.

Comment (itsayellow) [reply docs/troubleshooting.md:99]: Yes it works.  😄  Also I remember now that I had to change `python3` to `python` on Windows and I found this out by trying these commands in Windows.

**Comment by itsayellow** [comment]
@cs01, is this good to merge?

**Comment by cs01** [comment]
> @cs01, is this good to merge?

Yup looks good to me!

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**